xbmc/video/VideoDatabase.cpp

```diff
@@ -3934,6 +4039,46 @@ void CVideoDatabase::GetCast(int media_id, const std::string &media_type, std::v
   }
 }
 
+void CVideoDatabase::GetCast(const std::string& media_ids, const std::string &media_type, std::map<int, CVideoInfoTag>& details)
```

Comment:
Mainly to avoid looping in the map 4/5 times to always create the same string.. 
But you can also pass an empty map and it gets filled by the results of the query. For example in getEpisodesByWhere when I need the tvshow cast I pass an empty map.
I can easily switch to use the map ids. Both solutions aren't 100% nice.. Or you pass the string or you have to initialize the map and create the string multiple times.. You think it's better the 2nd one?